apps/mobile/src/hooks/use-safe-area-insets-style.ts

```diff
@@ -0,0 +1,51 @@
+import { useSafeAreaInsets, type Edge } from 'react-native-safe-area-context';
+import type { FlexStyle } from 'react-native';
+
+type ExtendedEdge = Edge | 'start' | 'end';
+
+const propertySuffixMap = {
```

Comment:
Check this naming convention
https://whatislove.dev/articles/javascript-naming-conventions-are-important/#:~:text=same%20naming%20conventions.-,Maps,-.%20Also%20known%20as